client/src/js/services/grid/GridGrouping.js

```diff
@@ -114,6 +116,13 @@ function GridGroupingService(uiGridGroupingConstants, $filter, Session, $timeout
     }
   }
 
+  function handleBatchSelection (){
```

Comment:
Why is this function necessary?  It seems like you only use it once.  To save on space, you could do this:
```js
function handleBatchSelection() {
  return this.gridApi.selection.getSelectedCount();
}
```
or, even better, do not create this function, like this:
```js
if (gridApi.selection) {
  gridApi.selection.on.rowSelectionChanged(null, selectAllGroupElements.bind(this));
  gridApi.selection.on.rowSelectionChangedBatch(null, selectAllGroupElements.bind(this));
}
```